### ISSUE TITLE
I have a question about a known issue.

### ISSUE THREAD

**Issue by Kyzoku** [soosgjr/Pull-Up-Panels#2]
I saw that you are aware of the issue of tabbed panels which dynamically change their height not working the best, but is there any way to mod in the ability to lock the location of the window?  The mod Minimal Display Bars has the ability to lock the positions of the status bars it adds and has the ability to specify an x and y position.  Sorry if its a dumb question, but I really like the idea of your mod.

**Comment by soosgjr** [comment]
No, that's a fine question. The challenge with that particular window is that by default all positions are stored as top-left corner coordinates, which is not ideal for windows that change their height. Exploring switching over to bottom-left corner coordinates is on my to-do list, but it's going to be a complex problem to solve, so there's currently no ETA on it.

Since the top-left is the default and the bottom-left requirement was introduced specifically for this mod, I doubt that there's a mod that would mitigate this problem.

A somewhat wonky workaround for the issue is undocking your most commonly used tab from the main tabbed window, but that requires editing an INI file and it's a bit buggy.
